SAU Model/ReadME.md

```diff
@@ -0,0 +1,28 @@
+# SAU DuckDB Creation Guide
+
+## Overview
+This repository contains scripts for generating DuckDB databases tailored for ARTIS model data:
+- **SAU_Duckdb_Creation_w_metadata.Rmd**: Generates a DuckDB database with metadata (non EML format).
+
+## Column Descriptions and Data Types
+Column specifications are sourced from the [ARTIS Database Tables](https://github.com/Seafood-Globalization-Lab/artis-model/wiki/ARTIS-Database-Tables). Metadata is based on these specifications. Discrepancies between table descriptions and source files are noted.
+
+## Notable Discrepancies
+- **Table: `consumption`**
```

Comment:
I updated all table documentation in the wiki pages based on your findings. I also added a column listing the SQL data type as well.